src/modules/EdgeDetect/EdgeUtils.js

```diff
@@ -198,8 +198,8 @@ var findMaxInMatrix = arr => Math.max(...arr.map(el => el.map(val => val ? val :
 // Applies the double threshold to the image.
 function doubleThreshold(pixels, highThresholdRatio, lowThresholdRatio, grads, strongEdgePixels, weakEdgePixels) {
 
-  const highThreshold = findMaxInMatrix(grads) * highThresholdRatio, // High Threshold relative to the strongest edge.
-    lowThreshold = highThreshold * lowThresholdRatio; // Low threshold relative to high threshold.
+  const highThreshold = findMaxInMatrix(grads) * highThresholdRatio, // High Threshold relative to the strongest edge
```

Comment:
T->t

Reply:
`High Threshold` is a name, a proper name for the High Threshold. It is not a generic high threshold, it is the Canny Edge Detection High Threshold.

Reply:
Then it is good !